tool/tctl/common/acl_command.go

```diff
@@ -35,6 +35,7 @@ import (
 	"github.com/gravitational/teleport/lib/auth/authclient"
 	"github.com/gravitational/teleport/lib/service/servicecfg"
 	"github.com/gravitational/teleport/lib/utils"
+	commonClient "github.com/gravitational/teleport/tool/tctl/common/client"
```

Comment:
```suggestion
	commonclient "github.com/gravitational/teleport/tool/tctl/common/client"
```

Package names (and their aliases) should generally be all lower case. (authclient, not authClient, testing/slogtest not testing/slogTest, etc).